apps/api/src/rest/index.ts

```diff
@@ -23,6 +24,7 @@ restRouter.route("/ping", pingRouter);
 restRouter.route("/search", searchRouter);
 restRouter.route("/websoc", websocRouter);
 restRouter.route("/week", weekRouter);
+restRouter.route("/study-rooms", studyRoomsRouter);
```

Comment:
REST resources should be camelCase
```suggestion
restRouter.route("/studyRooms", studyRoomsRouter);
```